src/config/graphical-session/kde.md

```diff
@@ -19,6 +19,19 @@ If you wish to start an X-based session from the console, use
 [startx](./xorg.md#startx) to run `startplasma-x11`. For a Wayland-based
 session, run `startplasma-wayland` directly.
 
+## SDDM
+
+Note that if you are configuring SDDM to [run under wayland][sddm-wayland],
+you will need to ensure the first virtual terminal is not consumed by
+`agetty`:
+
+```
+rm /var/service/agetty-tty1
```

Comment:
this is not the recommended way to stop agetty services, as this will be restored by an update to runit-void. instead, `touch /etv/sv/agetty-tty1/down`